ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/KsqlResourceFunctionalTest.java

```diff
@@ -158,9 +160,10 @@ public void shouldInsertIntoValuesForAvroTopic() throws Exception {
     TEST_HARNESS.getSchemaRegistryClient()
         .register(
             "books" + KsqlConstants.SCHEMA_REGISTRY_VALUE_SUFFIX,
-            SchemaUtil.buildAvroSchema(
+            AvroSchemas.getAvroSchema(
                 schema.valueSchema(),
-                "books" + KsqlConstants.SCHEMA_REGISTRY_VALUE_SUFFIX
+                "books_value",
```

Comment:
`SCHEMA_REGISTRY_VALUE_SUFFIX` has a dash, not an underscore. Does this discrepancy matter? (I see the name is used for the schema name and named map fields. Can't tell how these names are used, though.)